pkgs/development/python-modules/gower/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  setuptools,
+  fetchPypi,
+  scipy,
+  pandas,
+}:
+
+buildPythonPackage rec {
+
+  pname = "gower";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NN21FY8Oi/ugk9yga5+Ie9okSZjRCvKjrYx0pu+htfY=";
+  };
+
+  patchPhase = ''
+    rm pyproject.toml
```

Comment:
Why do you need this ? 
Setting `pyproject = true;` instead doesn't work ?

Reply:
Doing so results in the following error:

```
Creating a wheel...
* Getting build dependencies for wheel...
/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/dist.py:447: SetuptoolsDeprecationWarning: Invalid dash-separated options
!!

        ********************************************************************************
        Usage of dash-separated 'description-file' will not be supported in future
        versions. Please use the underscore name 'description_file' instead.

        This deprecation is overdue, please update your project and remove deprecated
        calls to avoid build errors in the future.

        See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
        ********************************************************************************

!!
  opt = self.warn_dash_deprecation(opt, section)
/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/config/_apply_pyprojecttoml.py:72: _MissingDynamic: `license` defined outside of `pyproject.toml` is ignored.
!!

        ********************************************************************************
        The following seems to be defined outside of `pyproject.toml`:

        `license = 'MIT'`

        According to the spec (see the link below), however, setuptools CANNOT
        consider this value unless `license` is listed as `dynamic`.

        https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-project-metadata-the-project-table

        To prevent this problem, you can list `license` under `dynamic` or alternatively
        remove the `[project]` table from your file and rely entirely on other means of
        configuration.
        ********************************************************************************

!!
  _handle_missing_dynamic(dist, project_table)
/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/config/_apply_pyprojecttoml.py:72: _MissingDynamic: `dependencies` defined outside of `pyproject.toml` is ignored.
!!

        ********************************************************************************
        The following seems to be defined outside of `pyproject.toml`:

        `dependencies = ['numpy', 'scipy']`

        According to the spec (see the link below), however, setuptools CANNOT
        consider this value unless `dependencies` is listed as `dynamic`.

        https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-project-metadata-the-project-table

        To prevent this problem, you can list `dependencies` under `dynamic` or alternatively
        remove the `[project]` table from your file and rely entirely on other means of
        configuration.
        ********************************************************************************

!!
  _handle_missing_dynamic(dist, project_table)
Traceback (most recent call last):
  File "/nix/store/mwvam3rmj6k3705z55griqzchlkjdajb-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
    main()
  File "/nix/store/mwvam3rmj6k3705z55griqzchlkjdajb-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_process.py", line 335, in main
    json_out['return_val'] = hook(**hook_input['kwargs'])
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/mwvam3rmj6k3705z55griqzchlkjdajb-python3.12-pyproject-hooks-1.0.0/lib/python3.12/site-packages/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
    return hook(config_settings)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 327, in get_requires_for_build_wheel
    return self._get_build_requires(config_settings, requirements=[])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 297, in _get_build_requires
    self.run_setup()
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/build_meta.py", line 313, in run_setup
    exec(code, locals())
  File "<string>", line 8, in <module>
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/__init__.py", line 108, in setup
    return distutils.core.setup(**attrs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/_distutils/core.py", line 158, in setup
    dist.parse_config_files()
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/dist.py", line 603, in parse_config_files
    pyprojecttoml.apply_configuration(self, filename, ignore_option_errors)
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/config/pyprojecttoml.py", line 71, in apply_configuration
    return _apply(dist, config, filepath)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 53, in apply
    _apply_project_table(dist, config, root_dir)
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 79, in _apply_project_table
    corresp(dist, value, root_dir)
  File "/nix/store/l6vlavpj6wzcihb1qmrh1xh8wh22cmfb-python3.12-setuptools-72.1.0/lib/python3.12/site-packages/setuptools/config/_apply_pyprojecttoml.py", line 181, in _license
    _set_config(dist, "license", val["text"])
                                 ~~~^^^^^^^^
KeyError: 'text'


```